### PR TITLE
Release 1.1.0

### PR DESCRIPTION
**Pull request by sergeysikachev**
- Developer(s): @sergeysikachev

- Peer Reviewer:

- Ticket: URL_HERE

- Project PR: URL_HERE

- Academy PR: URL_HERE (for feature documentation and if needed migration guides and project integration guide)

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department
- [ ] All new or heavily changed classes get an "A" rating (Scrutinizer), except Facades, Factories, QueryContainers and Tests

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraints         |
   :--------------------- | :------------------------ | :--------------------- |
   Payone               |       minor                 |                       |

#### Release Notes

- PayPal Express Checkout uses summary and success pages from standard checkout.
- Documentation describes possibility to add shipping select form to summary page.
- Requests and response logging have been added for debug purposes.
- CI scripts included
- Tests are migrated to new structure
- Fixed bug https://github.com/spryker/support/issues/147

-----------------------------------------

#### Bundle Payone

_**Major:** Backwards-compatible bug fix_

Def of done (by responsible developer):
- [ ] All changes are backward-compatible. Outdated code is marked as deprecated
- [ ] The change is isolated and not mixed with any cleanups or other changes
- [ ] New code fits to the architecture rules
- [ ] All new or changed business logic is covered by functional tests (in Zed business layer)

##### Change log

Features:

- PayPal Express Checkout uses summary and success pages from standard checkout.
- Documentation describes possibility to add shipping select form to summary page.
- Requests and response logging have been added for debug purposes.
- CI scripts included
- Tests are migrated to a new structure

Bugfixes:

 -  Fixed bug https://github.com/spryker/support/issues/147

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+notifications:
+  email: false
+
+sudo: required
+
+matrix:
+  fast_finish: true
+  include:
+     - php: "7.0"
```
Comment (dereuromark) [inline .travis.yml]: no more 7.0, only 7.1+ :)

File: composer.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "autoload": {
     "psr-0": {
       "SprykerEco": "src/",
-      "Function": "tests/"
+      "SprykerEcoTest": "tests/"
```
Comment (dereuromark) [inline composer.json]: No more psr0 please, only psr4

File: .travis.yml (hunk 1 above)
Comment (sergeysikachev) [reply .travis.yml]: Updated.

File: composer.json (hunk 2 above)
Comment (sergeysikachev) [reply composer.json]: Split into autoload and autoload-dev and changed psr-0 to par-4

File: composer.json [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
-    "psr-0": {
-      "SprykerEco": "src/",
-      "SprykerEcoTest": "tests/"
+    "psr-4": {
+      "SprykerEco\\": "src/"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "SprykerEcoTest\\": "tests/"
```
Comment (dereuromark) [inline composer.json]: This is not valid psr4, please check core how it is done (dev-master branch on spryker/spryker).

**Comment by dereuromark** [comment]
We recommend editorconfig file per repo in root, see https://github.com/spryker/spryker/blob/master/.editorconfig

File: README.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 # Payone Module
 
+[![Build Status](https://travis-ci.org/spryker-eco/payone.svg?branch=feature/ECO-508-ci-setup)](https://travis-ci.org/spryker-eco/payone)
```
Comment (dereuromark) [inline README.md]: I would set the master here always and never change it.

Comment (sergeysikachev) [reply README.md]: done!

**Pull request merged**